samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/author/TemplateLoadListener.java

```diff
@@ -152,6 +148,8 @@ public boolean loadAssessment(TemplateBean templateBean, String templateId)
         }
         if (aac.getLateHandling() != null)
           templateBean.setLateHandling(aac.getLateHandling().toString());
+        if (aac.getInstructorNotification() != null)
+          templateBean.setInstructorNotification(aac.getInstructorNotification().toString());
```

Comment:
I'd prefer to have curly braces on one-line if/else statements. I realize the surrounding code may not, but the precedent should be to include them.